src/ImageSharp/Formats/Webp/AlphaDecoder.cs

```diff
@@ -307,13 +311,50 @@ private void ExtractAlphaRows(Vp8LDecoder dec)
 
         private static void HorizontalUnfilter(Span<byte> prev, Span<byte> input, Span<byte> dst, int width)
         {
-            byte pred = (byte)(prev == null ? 0 : prev[0]);
+#if SUPPORTS_RUNTIME_INTRINSICS
+            if (Sse2.IsSupported)
+            {
+                dst[0] = (byte)(input[0] + (prev == null ? 0 : prev[0]));
```

Comment:
`prev` is a Span<byte>, so a valuetype and won't be `null`. You can skip that check.
Or is `prev.IsEmpty ? 0 : prev[0]` intended? But as a `private` method this could be asserted too.
```suggestion
                dst[0] = (byte)(input[0] + (prev.IsEmptyl ? 0 : prev[0]));
```